### PR TITLE
Fix up wrong information

### PR DESCRIPTION
**Pull request by shaedrich**
- Neither Open BD nor ACF ever supported that function
- Update docs state that function was added and deprecated in that version but gives not much more information

For discussion see: #760

### CODE REVIEW

**Comment by KamasamaK** [comment]
The same issue also applies to `getLocaleLanguage`.

**Comment by shaedrich** [comment]
You're right. Fixed it.

File: data/en/getlocalelanguage.json [hunk 1]
```diff
@@ -7,10 +7,7 @@
 	"description":"Gets the language from where the locale belongs to.",
 	"params": [],
 	"engines": {
-		"coldfusion": {"minimum_version":"", "notes":"", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-e-g/getlocaledisplayname.html"},
-		"lucee": {"minimum_version":"", "deprecated":"5", "notes":"Lucee5+ Deprecated in favor of getLocaleInfo()", "docs":"http://docs.lucee.org/reference/functions/getlocaledisplayname.html"},
-		"railo": {"minimum_version":"", "notes":"", "docs":"http://railodocs.org/index.cfm/function/getlocaledisplayname"},
-		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/function/getlocaledisplayname"}
+		"lucee": {"minimum_version":"", "added": "5.0", "deprecated":"5.0", "notes":"Lucee5+ Deprecated in favor of getLocaleInfo()", "docs":"http://docs.lucee.org/reference/functions/getlocaledisplayname.html"},
```
Comment (pfreitag) [inline data/en/getlocalelanguage.json]: Need to remove the trailing comma here for it be valid JSON

**Comment by KamasamaK** [comment]
The `docs` links are still pointing to a different function. Since the relevant docs don't actually exist, if they _have_ to point to a function, I think it makes more sense to use `getLocaleInfo`, but I believe they are not required. Also, they don't have a `removed` version, which is the main issue since it is not usable in the latest version.

File: data/en/getlocalelanguage.json (hunk 1 above)
Comment (shaedrich) [reply data/en/getlocalelanguage.json]: Oh, you're right.

**Comment by shaedrich** [comment]
In terms of "using" `getLocaleInfo` is of course the right and only choice. Noone ever said, you should use `getLocaleLanguage`.

@pfreitag To make a long story short, do we need it or do we not?

**Comment by KamasamaK** [comment]
@shaedrich You misunderstand. I was referring to what to use for the `docs` links, not what a developer uses.

**Comment by shaedrich** [comment]
@KamasamaK Oh, okay, sorry, I actually did. So link to from `getLocaleLanguage`'s cfdocs page to `getLocaleInfo`'s Lucee page?

**Comment by KamasamaK** [comment]
Yes, linking to `getLocaleInfo` was my suggestion over using `getLocaleDisplayName` which is what both are currently using. However, I still think it preferable to not include a `docs` link if no documentation for that function exists.

**Comment by shaedrich** [comment]
So do we remove the link change it to `getLocaleInfo`.

**Comment by pfreitag** [comment]
Sorry for delay in responding ... so if you remove the docs link it will not show the Lucee 5+ indicator on the top right... This may be ok however since it is deprecated right?

**Comment by shaedrich** [comment]
Well, makes sense. I do so.

**Comment by pfreitag** [comment]
Thx

**Pull request merged**